### PR TITLE
GEOPY-390

### PR DESCRIPTION
**Pull request by benk-mira**
**GEOPY-390 - Add a write_default_ui_json method in root of io module that finds all sub-modules and writes the a defaulted ui.json file for each.**
Requested fix to include peakfinder and octree files when run with use_initializers option

### CODE REVIEW

File: geoapps/io/write_default_uijson.py [hunk 1]
```diff
@@ -86,7 +86,9 @@ def write_default_uijson(path, use_initializers=False):
     for filename, params in filedict.items():
 
         if use_initializers:
-            if getattr(params, "forward_only", None) is not None:
+            if type(params) in [OctreeParams, PeakFinderParams]:
+                params.write_input_file(name=filename, path=path)
+            elif getattr(params, "forward_only", None) is not None:
```
Comment (domfournier) [inline geoapps/io/write_default_uijson.py:91]: I m still not getting the logic. When are we not writing with initializer?

Comment (domfournier) [reply geoapps/io/write_default_uijson.py:91]: Can we just set `default = not use_initializers`

File: geoapps/io/write_default_uijson.py [hunk 2]
```diff
@@ -84,12 +84,8 @@ def write_default_uijson(path, use_initializers=False):
     }
 
     for filename, params in filedict.items():
-
-        if use_initializers:
-            if getattr(params, "forward_only", None) is not None:
-                params.write_input_file(name=filename, path=path)
-        else:
-            params.write_input_file(name=filename, path=path, default=True)
+        default = False if use_initializers else True
```
Comment (domfournier) [inline geoapps/io/write_default_uijson.py:87]: Not to be picky, but that won't be mypy compliant. Convert to `not use_initializer` for simpler logical

**Review by domfournier — APPROVED** [review]

File: geoapps/io/write_default_uijson.py (hunk 2 above)
Comment (benk-mira) [reply geoapps/io/write_default_uijson.py:87]: Ternary's aren't mypy compliant?

Comment (domfournier) [reply geoapps/io/write_default_uijson.py]: either pylint or mypy, can't remember which but I had to replace every where on geoh5py.

**Pull request merged**